jest-puppeteer.config.js

```diff
@@ -3,6 +3,7 @@ const baseOptions = {
   server: {
     command: 'npm start',
     port: 8000,
+    launchTimeout: 30000,
```

Comment:
Why did you set the timeout value for launching the browser to 30 seconds?

Reply:
@sahsisunny For testing purpose. I was getting timeout error while testing. So set it to 30s.

Reply:
But in the GitHub workflow browser is not open it runs in CLI so we do not have to set the launch timeout for launching the browser.

Reply:
@sahsisunny I was getting below error:

```
Server has taken more than 10000ms to start.

☝️ You can set "server.launchTimeout" in jest-puppeteer.config.js
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
As mentioned in this error line ```You can set "server.launchTimeout" in jest-puppeteer.config.js``` I increased launchtimeout to 30s.
Also me and vinit referenced below link to find this solution:
https://github.com/argos-ci/jest-puppeteer/issues/89#issuecomment-400908716